templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Car</h1>
+  <p>Are you sure you want to delete the car "{{ object.model }}"?</p>
+
+  <form method="post">
+    {% csrf_token %}
+    <button type="submit" class="btn btn-danger">Delete</button>
+     <a href="{{ request.META.HTTP_REFERER }}" class="btn btn-secondary">Cancel</a>

```

Comment:
The `Cancel` button uses `HTTP_REFERER` to redirect to the previous page. Ensure that this approach works as expected, as `HTTP_REFERER` might not always be set, depending on the user's navigation path.